building_with_meson.md

```diff
@@ -9,8 +9,14 @@ into a problem._
 
 **Install build tools:** Use one of:
 
-- `mamba create -f environment.yml
-- `pip install -r build_requirements.txt  # also make sure you have pkg-config and the usual system dependencies for NumPy`
+- `mamba env create -f environment.yml && mamba activate numpy-dev
+
+- `python -m pip install -r build_requirements.txt
+  # also make sure you have pkg-config and the usual system dependencies for
+  # NumPy`
+
+Then install devpy:
+- `python -m pip install git+https://github.com/scientific-python/devpy`
```

Comment:
Let's leave this out of the docs, that should be a very temporary thing. `devpy` should be put on PyPI, so it can be added to the regular requirements files.